.github/workflows/go.yml

```diff
@@ -79,6 +79,47 @@ jobs:
         with:
           fail_ci_if_error: true
 
+  test-integration:
+    name: Integration Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.15"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Install binary
+        run: make install
+
+      - name: Get test log files
+        run: |
+          wget https://gist.githubusercontent.com/lukasmalkmus/9769d5e14171e36f05348e2e821a2e24/raw/b4bf910c8d60837a04fd734304762bf19b75eb36/logs.csv
+          wget https://gist.githubusercontent.com/lukasmalkmus/48e09c9b3e280ea31a4424fec1e54c98/raw/a7d73a0950b3e1b892d9656df6f96b4532ef6d94/logs.json
+          wget https://gist.githubusercontent.com/lukasmalkmus/72c058bb93d672f907eed0663fcce881/raw/0117f8ce988801d446878abde8c2d371de8db494/logs.ndjson
```

Comment:
Can we have them in a test stubs folder or something?

Reply:
The logs.ndjson file is huge (12MB), so I kinda prefer this. Is there a specific reason for having it in the repo?

Reply:
Oh yeah that's big. Can we generate it? I don't like the integration test being tied to your personal GitHub.

Reply:
I actually wanted to use some of our internal loggen stuff. Would we awesome if we could make that open source.

Reply:
You could also store 100 lines of ndjson and duplicate them n times for the test

Reply:
Which makes absolute sense tbh.